server/templates/pages/src/update_firefox.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html dir="{{ lang_dir }}" lang="{{ lang }}">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>{{#t}}Firefox Accounts{{/t}}</title>
+        <meta name="description" content="">
+        <meta name="referrer" content="origin">
+        <meta name="robots" content="noindex,nofollow">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=2.0, user-scalable=yes" />
+
+        <!--iOS Smart Banner-->
+        <meta name="apple-itunes-app" content="app-id=989804926, affiliate-data=ct=smartbanner-fxa">
+
+        <!-- build:css(.tmp) /styles/main.css -->
+        <link rel="stylesheet" href="/styles/main.css">
+        <!-- endbuild -->
+
+        <script type="text/javascript" src="{{ bundlePath }}/head.bundle.js"></script>
+    </head>
+    <body class="static">
+      <div id="fox-logo"></div>
+      <div id="stage">
+        <div id="main-content" class="card">
+          <header>
+            <h1 id="fxa-update-firefox-header">{{#t}}Firefox update required{{/t}}</h1>
+          </header>
+
+          <section>
+            <p>
+              {{#t}}Firefox Accounts makes use of features that are not supported in your version of Firefox. Please download and install the latest version of Firefox to continue.{{/t}}
+            </p>
+            <div class="button-row">
+              <a class="button primary-button" href="https://www.mozilla.org/firefox/download/thanks/">{{#t}}Download latest{{/t}}</a>
```

Comment:
It seems reasonable to log users that click on this, but that'd require importing the JS bundle, which I was hoping to avoid. We could point to a route on our server, log the click, and then redirect to download firefox endpoint? More trouble than it's worth?

Reply:
> Don't they have metrics on the download endpoint to see where people show up from?

I assume so, the downside is we'd have to ask them instead of just looking in Amplitude.

Reply:
true!